src/components/manifold-plan-table/manifold-plan-table.tsx

```diff
@@ -48,20 +48,35 @@ interface PlanFeatures {
 
 // state
 type UserValue = string | number | boolean;
+type FeatureMap = { [featureLabel: string]: UserValue };
 interface UserSelection {
-  [planID: string]: { [featureLabel: string]: UserValue };
+  [planID: string]: FeatureMap;
+}
+
+// event
+
+interface PlanTableInit {
+  userSelection: any;
+}
+
+interface PlanTableEvent {
+  planID: string;
+  planDisplayName: string;
+  userSelection: FeatureMap;
 }
 
 @Component({ tag: 'manifold-plan-table' })
 export class ManifoldPlanTable {
   @Element() el: HTMLElement;
-  @Event() CTAClick: EventEmitter;
+  @Event({ bubbles: false }) ctaClick: EventEmitter<PlanTableEvent>;
+  @Event({ bubbles: false }) init: EventEmitter<PlanTableInit>;
+  @Event({ bubbles: false }) update: EventEmitter<PlanTableEvent>;
   // Passed product ID to the graphql endpoint
   @Prop() productId?: string;
   // Passed client ID header to the graphql calls
   @Prop() clientId?: string = '';
   // Base url for buttons
-  @Prop() baseUrl?: string = '/signup';
+  @Prop() baseUrl?: string;
```

Comment:
`baseUrl` default was removed…

Reply:
I was thinking recently that I really expect the event to be more commonly used than the URL, assuming both are equally visible in the API. And then I thought that maybe we can sort of test for that by adding a field to the analytics event that indicates which version the provider chose. Then if it became clear that providing a `baseUrl` is never really done in practice, we could eventually drop support for it so we don't have to always handle these in two ways.